qemu/tests/qemu_guest_agent.py

```diff
@@ -232,8 +232,42 @@ def get_mdl_tag_build_status(get_mdl_tag_cmd):
                                         timeout=query_timeout
                                         ).strip().decode()
         logging.info("Qemu-guest-agent rpm pkg url is %s" % rpm_url)
+        self.rpm_url = rpm_url
```

Comment:
1. This part should be reorganized so here I guess you can not utilize it this way before the reorganization.
2. If I understand correctly, here we have two different packages, `qemu-guest-agent-container` and `qemu-guest-agent`, it should not be united to a single variable.

Reply:
yes, qemu-guest-container part I should change code logic,because the way query rpm pkg tag from brew is related with another bug that move code logic to internal,so I can fix this case now.